### PR TITLE
run/config: update outpus with new data model

### PR DESCRIPTION
**Pull request by ppanero**
Closes https://github.com/inveniosoftware/docs-invenio-rdm/issues/16

### CODE REVIEW

File: docs/develop/config.md [hunk 1]
```diff
@@ -36,22 +36,60 @@ That's it configuration-wise. If we try to create a record through the API, your
...
 ``` console
-$ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '
-{
-    "access": {
-        "metadata_restricted": "false",
-        "files_restricted": "false"
+$ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '{
+	"_access": {
+		"metadata_restricted": "false",
+		"files_restricted": "false"
```
Comment (fenekku) [inline docs/develop/config.md]: Booleans should not be quoted, right? Should be `"metadata_restricted": false` and so on.

File: docs/develop/config.md [hunk 2]
```diff
@@ -88,22 +126,60 @@ Note that this user will have ID 2.
...
-    "description": "A Holiday Record description",
-    "owners": [1],
-    "publication_date": "25/12/2019",
-    "resource_type": {
-        "type": "Dataset",
-        "subtype": "dataset"
+$ curl -k -XPOST -H "Authorization:Bearer sHHq1K9y7a2v5doKDRSFmSFOxa1tZDHFcbs31npaxm1sFEt27yomLMt0ynkl" -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '{
+    "_access": {
+		"metadata_restricted": "false",
+		"files_restricted": "false"
```
Comment (fenekku) [inline docs/develop/config.md]: same here

File: docs/develop/run.md [hunk 3]
```diff
@@ -48,37 +48,163 @@ $ curl -k -XGET https://localhost:5000/api/records/ | python3 -m json.tool
...
-                "id": "s2pwq-mzw48",
+                "created": "2020-02-25T15:54:52.127129+00:00",
+                "updated": "2020-02-25T15:54:52.127134+00:00",
+                "revision": 0,
+                "id": "zgxnf-z7n12",
                 "links": {
-                    "files": "https://localhost:5000/api/records/s2pwq-mzw48/files",
-                    "self": "https://localhost:5000/api/records/s2pwq-mzw48"
+                "files": "https://localhost/api/records/zgxnf-z7n12/files",
+                "self": "https://localhost/api/records/zgxnf-z7n12"
```
Comment (fenekku) [inline docs/develop/run.md]: Let's keep indentation and `:5000`

File: docs/develop/run.md [hunk 4]
```diff
@@ -48,37 +48,163 @@ $ curl -k -XGET https://localhost:5000/api/records/ | python3 -m json.tool
...
+                    "contact": "info@inveniosoftware.org",
                     "contributors": [
                         {
-                            "name": "Jeanne Jones"
+                        "affiliations": [
+                            {
+                            "identifier": "entity-one",
+                            "name": "Doyle, Miller and Williams",
+                            "scheme": "entity-id-scheme"
+                            }
```
Comment (fenekku) [inline docs/develop/run.md]: I think something weird happened with the indentation across this file. We should re-indent appropriately.

File: docs/develop/run.md [hunk 5]
```diff
@@ -98,29 +224,67 @@ $ curl -k -XGET https://localhost:5000/api/records/ | jq .
...
-    "description": "Photo-analysis of doge in crystalline structures",
-    "owners": [1],
-    "publication_date": "05/11/2019",
-    "resource_type": {
-        "type": "image",
-        "subtype": "photo"
+$ curl -k -XPOST -H "Content-Type: application/json" https://localhost/api/records/ -d '{
+    "_access": {
+		"metadata_restricted": "false",
+		"files_restricted": "false"
```
Comment (fenekku) [inline docs/develop/run.md]: same note about booleans

File: docs/develop/run.md [hunk 6]
```diff
@@ -98,29 +224,67 @@ $ curl -k -XGET https://localhost:5000/api/records/ | jq .
...
+	},
+	"_owners": [1],
+	"_created_by": 1,
+	"access_right": "open",
+	"resource_type": {
+		"type": "image",
+		"subtype": "photo"
```
Comment (fenekku) [inline docs/develop/run.md]: Same note about indentation.

File: docs/develop/run.md [hunk 7]
```diff
@@ -98,29 +224,67 @@ $ curl -k -XGET https://localhost:5000/api/records/ | jq .
...
-    },
-    "access_right": "open",
-    "contributors": [{"name": "Jon Doe"}, {"name": "Ian Matyssik"}],
-    "description": "Photo-analysis of doge in crystalline structures",
-    "owners": [1],
-    "publication_date": "05/11/2019",
-    "resource_type": {
-        "type": "image",
-        "subtype": "photo"
+$ curl -k -XPOST -H "Content-Type: application/json" https://localhost/api/records/ -d '{
```
Comment (fenekku) [inline docs/develop/run.md]:
```suggestion
$ curl -k -XPOST -H "Content-Type: application/json" https://localhost:5000/api/records/ -d '{
```

File: docs/develop/config.md (hunk 1 above)
Comment (ppanero) [reply docs/develop/config.md]: True, thanks!

File: docs/develop/run.md (hunk 6 above)
Comment (ppanero) [reply docs/develop/run.md]: damn this one was showing correctly on my pc hehe

**Pull request merged**